log/keyvalue.go

```diff
@@ -267,6 +270,16 @@ func (v Value) Equal(w Value) bool {
 	}
 }
 
+func sortMap(m []KeyValue) []KeyValue {
+	sm := make([]KeyValue, len(m))
+	copy(sm, m)
+	slices.SortFunc(sm, func(a, b KeyValue) int {
+		return cmp.Compare(a.Key, b.Key)
+	})
+
+	return sm
```

Comment:
We can use https://pkg.go.dev/slices#Clone. E.g.

```suggestion
	m = slices.Clone(m)
	slices.SortFunc(m, func(a, b KeyValue) int {
		return cmp.Compare(a.Key, b.Key)
	})
	return m
```